api/apps/api/src/modules/specification/determine-features/application/specification.repository.ts

```diff
@@ -0,0 +1,7 @@
+import { Specification } from '../../specification';
```

Comment:
the application is chunked too much in my opinion

this import probably leads to circular dependency

it should be:

```
specification/domain
specification/application/determine-feature
```
and so on

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/417/files via https://github.com/Vizzuality/marxan-cloud/pull/417/commits/94946b083a4e324560dcb1e72bdd94f5f4b46799